packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -53,6 +32,135 @@ describe('App: Settings', () => {
     cy.wait('@ReconfigureProject')
   })
 
+  describe('Project Settings', () => {
+    it('shows the projectId section when there is a projectId', () => {
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.findByText('Project ID').should('be.visible')
+    })
+
+    it('shows the Record Keys section', () => {
+      cy.loginUser()
+
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.findByText('Record Key').should('be.visible')
+    })
+
+    it('obfuscates each record key and has a button to reveal the key', () => {
+      cy.loginUser()
+
+      cy.visitApp()
+      cy.findByText('Settings').click()
+      cy.findByText('Project Settings').click()
+      cy.get('[data-cy="record-key"]').should('contain', '***')
```

Comment:
Seeing a failure here when running this locally. Not sure if you've seen that happen? But I'd like to avoid a new flake if possible

Reply:
hmm that's strange i'm not able to reproduce a failure locally for some reason...

Reply:
@davidmunechika I pulled fresh and rebuilt everything and this is working for me now. I've tried several times so I think we're all good.